<div class="vf-grid vf-grid__col-4">
  <div class="vf-grid__col--span-3">
    <section class="vf-content" style="margin-bottom: 8rem">
      <div class="vf-grid vf-grid__col-4">
        <h1 class="vf-text vf-text-heading--4">Aims</h1>
        <div class="vf-grid__col--span-3">
          <p>
            The Human Cell Atlas (HCA) Project Catalogue by EMBL-EBI aims to
            provide a comprehensive list of all the cellular resolution data
            generated by and available to the HCA community.
          </p>
          <p>
            By providing links to the publications and the resources where the
            data is located, the catalogue allows quick access to all newly
            generated data in the HCA community, and accelerates collaboration
            in the building of the atlas.
          </p>

          <p>
            The list is curated by the HCA Data Coordination Platform wrangling
            team at EMBL-EBI, but is open for contribution by anyone who works
            within the single cell technology field.
          </p>
          <p>
            To send a suggestion for the Catalogue, use the
            <a routerLink="/add-project">Suggest a project </a>
            form.
          </p>
        </div>

        <h1 class="vf-text vf-text-heading--4">Eligibility Criteria</h1>
        <div class="vf-grid__col--span-3">
          <p>
            There are three main categories of data considered eligible for the
            Human Cell Atlas, and respectively, the HCA Project Catalogue.
          </p>
          <p>
            <b
              >Data directly contributed to the HCA Data Coordination
              Platform</b
            >
          </p>
          <p>To be eligible for this category, datasets must:</p>
          <ul>
            <li>
              Be contributed to the
              <a href="https://contribute.data.humancellatlas.org/">
                Human Cell Atlas Data Coordination Platform</a
              >
              by the data owner, with support of the HCA DCP data wranglers.
            </li>
            <li>
              Have either a pre-print publication referencing data that meets
              the criteria for published data (see below), or a supporting
              justification as to why it should be included in the HCA.
            </li>
            <li>
              Be available in raw fastq format and/or as a gene expression
              matrix (raw counts or processed counts). Data from multiple
              samples must be demultiplexed (each fastq file contains data from
              a single sample) to be considered eligible. If only multiplexed
              data is available, gene expression matrices will still be
              considered eligible.
            </li>
          </ul>

          <p><b>Data from official Human Cell Atlas publications</b></p>
          <p>To be eligible for this category, datasets must:</p>

          <ul>
            <li>
              Support a publication that has been reviewed by the HCA
              publication committee and added to the official
              <a href="https://www.humancellatlas.org/publications/"
                >HCA publications list</a
              >
            </li>
            <li>
              Be available in raw fastq format and/or as a gene expression
              matrix (raw counts or processed counts). Data from multiple
              samples must be demultiplexed (each fastq file contains data from
              a single sample) to be considered eligible. If only multiplexed
              data is available, gene expression matrices will still be
              considered eligible.
            </li>
          </ul>

          <p><b>Other published data</b></p>
          <p>To be eligible for this category, datasets must:</p>
          <ul>
            <li>
              Be referenced in a peer-reviewed journal or be available as
              pre-print
            </li>
            <li>
              Contain at least some samples derived from healthy human donors
            </li>
            <li>Be derived from a recognised single cell technology.</li>
            <li>
              Be available in raw fastq format and/or as a gene expression
              matrix (raw counts or processed counts). Data from multiple
              samples must be demultiplexed (each fastq file contains data from
              a single sample) to be considered eligible. If only multiplexed
              data is available, gene expression matrices will still be
              considered eligible.
            </li>
          </ul>
        </div>

        <h1 class="vf-text vf-text-heading--4">Prioritisation</h1>
        <div class="vf-grid__col--span-3">
          <ol>
            <li>
              <b
                >Data directly contributed to the HCA Data Coordination
                Platform</b
              >
              <ul>
                <li>
                  From co-funded collaborators (e.g. WSSS, MRC, HCA:Organoid,
                  Seed Networks), then
                </li>
                <li>First come first serve</li>
              </ul>
            </li>

            <li>
              <b>Data from official Human Cell Atlas publications</b>
              <ul>
                <li>Newest first, oldest last</li>
                <li>Published Data on primary tissues</li>
                <li>
                  Poorly represented donor sample locations first (i.e. US,
                  European donors will be actively down-prioritised). Once data
                  from all possible locations is captured, then
                </li>
                <li>
                  Organs with least coverage first, organs with most coverage
                  last
                </li>
              </ul>
            </li>
            <li>
              <b>All other eligible published data</b>
              scored by a combination of:
              <ul>
                <li>Organ coverage in the HCA, and</li>
                <li>Cell count size, and</li>
                <li>Commonality of technology</li>
              </ul>
            </li>
          </ol>
        </div>

        <h1 class="vf-text vf-text-heading--4">Data sources</h1>

        <div class="vf-grid__col--span-3">
          <p>
            HCA Project Catalogue contains projects which are selected and
            curated by the HCA Data Coordination Platform wranglers, from these
            main sources:
          </p>

          <ul>
            <li>The official HCA publications list</li>
            <li>The HCA Data Portal</li>
            <li>
              Projects are funded to contribute data to the HCA including:
              <ul>
                <li>Wellcome Strategic Science Support Award</li>
                <li>MRC Developmental Cell Atlas</li>
                <li>Organoid Horizon 2020 award</li>
                <li>CZI seed network funding</li>
                <li>CZI biological network funding</li>
              </ul>
            </li>
            <li>
              Established data archives
              <ul>
                <li>GEO</li>
                <li>ArrayExpress</li>
              </ul>
            </li>
            <li>
              Specialised databases
              <ul>
                <li>Single Cell Expression Atlas</li>
                <li>
                  <a
                    href="https://www.nxn.se/single-cell-studies"
                    target="_blank"
                    >The Single cell studies database</a
                  >, maintained by Valentine Svensson
                </li>
                <li>
                  <a href="https://www.covid19cellatlas.org" target="_blank"
                    >The COVID19 Cell Atlas</a
                  >
                </li>
              </ul>
            </li>
            <li>Suggestions from the HCA community (that means you!)</li>
          </ul>
        </div>

        <h1 class="vf-text vf-text-heading--4">Contribute a project</h1>

        <div class="vf-grid__col--span-3">
          <p>
            This catalogue is created for and by the HCA community and aims to
            provide quick and easy access to suitable data. Help us build a
            comprehensive catalogue by letting us know if we have missed a
            project.
          </p>
          <p>
            The Data wrangling team can support the inclusion of these types of
            projects:
          </p>
          <ul>
            <li>
              A project which is already published and the data is available
              elsewhere. The data must be available in raw fastq format and/or
              as a gene expression matrix (raw counts or processed counts). Data
              from multiple samples must be demultiplexed to be considered
              eligible. If only multiplexed data is available, gene expression
              matrices will still be considered eligible.
            </li>
            <li>
              A project, which is in the process of publication, and data is
              pending or needs an accession
            </li>
            <li>
              A project, which is under way, but there is no publication or data
              available yet. (The Data Wrangling team can support you in
              gathering and organising its metadata)
            </li>
          </ul>

          <a
            class="vf-button vf-button--primary vf-button--sm"
            routerLink="/add-project"
            >Suggest a project
          </a>
        </div>

        <h1 class="vf-text vf-text-heading--4">Join the community</h1>

        <div class="vf-grid__col--span-3">
          <p>
            To join the HCA community,
            <a href="https://www.humancellatlas.org/join-hca/" target="_blank"
              >register your interest here</a
            >
            to stay up to date with community happenings and events.
          </p>
        </div>
      </div>
    </section>
  </div>
</div>
